privacyidea/lib/eventhandler/base.py

```diff
@@ -140,6 +141,11 @@ def conditions(cls):
                           "not exist in the userstore anymore."),
                 "value": ("True", "False")
             },
+            CONDITION.TOKEN_IS_EXPIRED: {
+                "type": "str",
+                "desc": _("The token has an expired validity period"),
```

Comment:
I will go with the simplicity for now ;-)
I will go with the condition name "token_validity_period", and value true/false, and will also update the test cases.